spruce/Dockerfile

```diff
@@ -1,6 +1,11 @@
 FROM golang:1.5.1-alpine
 
-RUN apk update
-RUN apk add git \
+ENV SPRUCE_VERSION 0.12.0
```

Comment:
Because I only want to pin the version, not upgrade. I will upgrade in a different PR